hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/filedrops/V2FileDrops.java

```diff
@@ -0,0 +1,101 @@
+package ca.bc.gov.hlth.hnsecure.filedrops;
+
+
+import static java.nio.file.StandardOpenOption.CREATE;
+
+import java.io.BufferedOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Handler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ca.bc.gov.hlth.hnsecure.parsing.Util;
+
+/**
+ * This implementation logs each HL7v2 request/response message in a file in its raw/text form.
+ * file name format:{messageid}-{messagetype}-{facilityid}-{messagedate}-{request/response}.txt
+ * @author anumeha.srivastava
+ *
+ */
+public class V2FileDrops {
+	
+	private static final Logger logger = LoggerFactory.getLogger(V2FileDrops.class);
+	
+	public final static String REQUEST_FILE = "request.txt";
+	public final static String RESPONSE_FILE = "response.txt";
```

Comment:
It's best practice to have "static final" as opposed to "final static". See https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers.

Reply:
Typo mistake. corrected